### ISSUE TITLE
Google Maps loads but produces a blank page in 0.7

### ISSUE THREAD

**Issue by jbmonroe** [brave/browser-laptop#493]
Title pretty much says it all.  I could see the source download just fine, but something prevents Maps from drawing anything.  I hit https://www.google.com/maps with no additional data.

**Comment by bridiver** [comment]
I'm not able to replicate this, do you have the latest build from brave.com? The browser currently just displays an empty page if there is a network issue so that could be the problem. That's on our list of things that need to be completed soon.

**Comment by jbmonroe** [comment]
I thought I had the latest version--Brave downloaded it for me and I ran the installer.  I pulled the latest Dev release from the website and tried to replicate, and it does work as you said.  It would have been an interesting network glitch because I'd been using Maps just a minute before in Chrome, and I wasn't seeing network problems anywhere else.  (Nor was my family complaining.)  Since it isn't reproducible, I'll withdraw it.
